yucca/training/trainers/YuccaLightningManager.py

```diff
@@ -159,20 +160,23 @@ def run_training(self):
     def predict_folder(
         self,
         input_folder,
+        disable_tta: bool = False,
         output_folder: str = yucca_results,
         save_softmax=False,
     ):
         self.initialize(
             stage="predict",
+            disable_tta=disable_tta,
             pred_data_dir=input_folder,
             segmentation_output_dir=output_folder,
             save_softmax=save_softmax,
         )
-        self.trainer.predict(
-            model=self.model_module,
-            dataloaders=self.data_module,
-            ckpt_path=self.ckpt_path,
-        )
+        with torch.inference_mode():
```

Comment:
Shouldn't lightning handle this?

Reply:
https://github.com/Lightning-AI/lightning/issues/11530 It doesn't seem to be the case, but I honestly couldn't tell for sure so I added it in to be 100% sure we don't accidentally train during inference